lib/types/index.js

```diff
@@ -431,6 +421,21 @@ export type Feed = {
   versionCount: number
 }
 
+export type FeedSourceSummary = {
```

Comment:
Maybe move the declaration for `FeedSourceSummary` to above the one for `FeedSource` and that could take care of the `name` duplication issue?

Reply:
Sure, that's updated in https://github.com/ibi-group/datatools-ui/pull/981/commits/f23755a3247b1cf848624d1298ce34adffd9717e